config/routes.rb

```diff
@@ -12,6 +15,8 @@
     match "/(*path)" => redirect { |_, req| "#{Settings.base_url}#{req.fullpath}" }, via: %i[get post put]
   end
 
+  mount Sidekiq::Web, at: "/sidekiq", constraints: SystemAdminConstraint.new
```

Comment:
This should sit in the `publish` route section right?

> Publish has had sidekiq setup for a while but we have never enabled the web interface to be available. This PR does that so we can diagnose jobs. Using the same constraint from Register.

Reply:
agreed, moved